s3sync.go

```diff
@@ -222,6 +246,27 @@ func (m *Manager) syncS3ToLocal(ctx context.Context, chJob chan func(), sourcePa
 	return errs.ErrOrNil()
 }
 
+func (m *Manager) copy(ctx context.Context, file *fileInfo, sourcePath *s3Path, destPath *s3Path) error {
```

Comment:
The func name would be better to indicate it's s3 to s3

Reply:
thank you! Updated in commit 4addce6